bubble_sort/Java/BubbleSort.java

```diff
@@ -0,0 +1,58 @@
+/*
+* I have converted the code of bubble sort written in C++ into Java with some little modifications,
+* C++ code: https://github.com/NITSkmOS/Algorithms/blob/master/bubble_sort/Cpp/bubble_sort.cpp
+* */
+
+import java.util.ArrayList;
+
+public class BubbleSort
+{
+    private ArrayList<Integer> arr;
+
+    public BubbleSort(ArrayList<Integer> arr)
+    {
+        this.arr = arr;
+        bubbleSort();       // Main bubble sort algorithm is called
+    }
+
+    // Wrapper class is used so that values are changed inside the function
+    private void swap(int j)
+    {
+        int temp = arr.get(j);
+	    arr.set(j, arr.get(j + 1));
+	    arr.set(j+1, temp);
+    }
+
+    private void bubbleSort()
+    {
+        int i, j;
+        Boolean swapped;
+        for (i = 0; i < arr.size(); i++)
+        {
+            swapped = false;
+            for (j = 0; j < arr.size()-i-1; j++)
+            {
+                if (arr.get(j) > arr.get(j+1))
+                {
+                    swap(j);
+                    swapped = true;
+                }
+            }
+
+            // If no two elements were swapped by inner loop, then break
+            if (!swapped)
+                break;
+        }
+    }
+
+    void printSortedList()
+    {
+        System.out.println("\nNumbers after applying bubble sort algorithm are: ");
+        int i;
+        for (i = 0; i < arr.size(); i++)
+        {
+            System.out.print(arr.get(i) + " ");
+        }
+        System.out.println();
+    }
+}
```

Comment:
Line contains following spacing inconsistencies:
- No newline at EOF.

*Origin: SpaceConsistencyBear, Section: `all.pyjava`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpqa3_fkic/bubble_sort/Java/BubbleSort.java
+++ b/tmp/tmpqa3_fkic/bubble_sort/Java/BubbleSort.java
@@ -55,4 +55,4 @@
         }
         System.out.println();
     }
-}+}
```

Reply:
Have you applied the changes ? or Do I have to submit a new pull request ?

Reply:
I have make a new directory and files and pushed it,
I guess issue is not assigned yet.

Reply:
@saboorElahi, You neither asked for the assignment nor used issue link in your PR. How will I know the issue?
Ping on gitter if I forget https://gitter.im/NITSkmOS/algo